Movie_MVP/Movie_MVP/DataStorageService/Repository/CoreDataFilmsRepository.swift

```diff
@@ -0,0 +1,37 @@
+// CoreDataFilmsRepository.swift
+// Copyright © RoadMap. All rights reserved.
+
+import CoreData
+
+final class CoreDataFilmsRepository: Database {
```

Comment:
```suggestion
final class CoreDataFilmsRepository: Database {
```
дублирование кода ради 2 х разных строк сделай 1 универсальную из 2х